src/vue-components/inflections-table-wide.vue

```diff
@@ -2,7 +2,7 @@
     <div v-if="!view.isImplemented" class="alpheios-inflections__not-impl-msg">
         {{messages.INFLECT_MSG_TABLE_NOT_IMPLEMENTED}}
     </div>
-    <div v-else-if="view.wideView && !view.isEmpty">
+    <div v-else>
```

Comment:
we don't need the test on isEmpty?

Reply:
We don't need it here because we always want to show a title now. After optimizations we show titles for not-rendered-yet tables of inflection browser. It's a click on a title that makes it to render. That's a strategy different from before.

I don't remember exactly but I believe we used it before to filter out results of incorrect lexical queries or something like that and now we have some other checks for that. During my test I was not able to reproduce a situation when `isEmpty` check would be needed. However, if you know a test case that would require that, please let me know.

Reply:
ok, no that makes sense.